cmd/config-translator/translator_test.go

```diff
@@ -19,14 +19,14 @@ import (
 func checkIfSchemaValidateAsExpected(t *testing.T, jsonInputPath string, shouldSuccess bool, expectedErrorMap map[string]int) {
 	actualErrorMap := make(map[string]int)
 
-	jsonInputMap, err := util.GetJsonMapFromFile(jsonInputPath)
-	if err != nil {
-		t.Fatalf("Failed to get json map from %v with error: %v", jsonInputPath, err)
+	jsonInputMap, e := util.GetJsonMapFromFile(jsonInputPath)
+	if e != nil {
+		t.Fatalf("Failed to get json map from %v with error: %v", jsonInputPath, e)
```

Comment:
This can be simplified to:
```suggestion
	assert.NoError(t, e)
```

There is a lot of code like this and I wanted to clean up our test code myself, so I'm fine with leaving this as is in your PR because I'll just update it in mine.